### ISSUE TITLE
Appropriate way to freeze the inception graph.

### ISSUE THREAD

**Issue by akash-47** [tensorflow/models#866]
Model name : Inception

I am trying to classify images in my custom dataset.
All the steps in the readme work quite well for me.

However, I am not able to freeze the graph in order to use the model for prediction.
The issue is with `output_node_name` not being able to find the output layer. 

The inception_eval builds the graph and calculates predictions using logits. I used it for creating a softmax layer and then tried to calculate prediction using the [softmax](https://www.tensorflow.org/api_docs/python/nn/classification#softmax) classifier. It has not worked out for me.

Is there something I am missing here?
What might be an appropriate way to use the pre-trained model for predictions ?

**Comment by jart** [comment]
Thanks for reaching out. We use this issue tracker for bugs and feature requests. We recommend going to [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) for community support.
